ts/Dashboards/Board.ts

```diff
@@ -178,17 +178,21 @@ class Board implements Serializable<Board, Board.JSON> {
 
         // Init edit mode.
         if (this.guiEnabled) {
-            if (
-                EditMode && !(
-                    this.options.editMode &&
-                    !this.options.editMode.enabled
-                )
-            ) {
-                this.editMode = new EditMode(this, this.options.editMode);
+            if (!Dashboards.EditMode) {
+                throw new Error('Missing editmode.js module');
```

Comment:
If you are using layouts, it is unclear if we should call it `editmode`... The `layout` sounds more intuitive.